user_profile/templates/mi_fuego/my_tickets/index.html

```diff
@@ -84,7 +86,7 @@
     {% if holding_tickets|add:shared_tickets > 0 %}
         <li class="nav-item">
             <a class="nav-link{% if nav_secondary == "transferable_tickets" %} active{% endif %}"
-               href="{% url 'transferable_tickets' %}">Bonos para transferir ({{ holding_tickets }}) </a>
+               href="{% url 'transferable_tickets' %}">Bonos adicionales ({{ holding_tickets }}) </a>
```

Comment:
I renamed "Bonos para transferir" to "Bonos adicionales" because we agreed we'll allow one person to buy more tickets so that for 5 tickets people don't need to create 5 accounts with email/phone verification. This allows showing them the "additional tickets", but it hides the mentioning that these are "tickets to transfer", because they are not.